### PR TITLE
Fixing/Upgrade responsive UI/UX

### PR DESCRIPTION
**Pull request by hgvo9**
### Checklist
☑️ TDD Security.
☑️ Minimal Testing.
✅ Accurate assignees / reviewers.
- - -
### Description
Fixing the different issues that we had in the design of the map for different smartphones, now we have the exact same design no matter the screen dimensions.

Screenshot 2021-11-28 at 15 10 32

### CODE REVIEW

File: EntryPoint.tsx [hunk 1]
```diff
@@ -27,19 +29,30 @@ const EntryPoint: React.FC = () => {
...
+            if (route.name === "Home")
+              iconName = faLocationArrow
+            if (route.name === "Profile")
+              iconName = faStore
+            if (route.name === "Saved Places")
+              iconName = faMapPin
+            
+            return <FontAwesomeIcon icon={iconName} color={"#1E90FF"} size={14}/>
 
+        //     let iconName;
```
Comment (louischristner) [inline EntryPoint.tsx:45]: You can remove commented code

File: src/components/Home.tsx [hunk 2]
```diff
@@ -189,53 +194,96 @@ export const HomeComponent = ({getNearestSafe, count, setCount, goToSafeplace, s
...
                                                 key={index} 
-                                                style={{zIndex: 9999, marginBottom: 10}}
-                                                >
-                                                <Text 
+                                                style={{zIndex: 999}}
                                                 >
-                                                    {place.description}
-                                                </Text>
+                                                    <Text 
+                                                    >
```
Comment (louischristner) [inline src/components/Home.tsx:244]: You can combine these two lines in one

File: src/containers/Home.tsx [hunk 3]
```diff
@@ -36,6 +36,7 @@ export const Home = (): JSX.Element => {
   const [navigationMode, setNavigationMode] = useState<boolean>(false);
   const navigation = useNavigation();
   const [count, setCount] = useState<number>(0);
+  const [heading, setHeading] = useState<LocationHeadingObject>();
```
Comment (louischristner) [inline src/containers/Home.tsx:39]: If you don't use these you can remove it

File: src/containers/Home.tsx [hunk 4]
```diff
@@ -109,6 +109,14 @@ export const Home = (): JSX.Element => {
...
   })();
 
+  // let unsubscribeHeadingDevice = (async (): Promise<Location.LocationSubscription> => {
+  //   return await Location.watchHeadingAsync((heading) => {
+  //       setHeading(heading);
+  //       console.log(heading);
+  //   });
+  // });
+
+    // unsubscribeHeadingDevice();
```
Comment (louischristner) [inline src/containers/Home.tsx:119]: Same hear for commented lines

File: EntryPoint.tsx [hunk 5]
```diff
@@ -10,6 +10,8 @@ import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
 // import Ionicons from 'react-native-vector-icons/Ionicons';
 import Toast from 'react-native-toast-message';
 import { useAppSelector } from './src/utils/hooks';
+import { faDirections, faLocationArrow, faCircle, faMapPin, faStore } from '@fortawesome/free-solid-svg-icons'
```
Comment (louischristner) [inline EntryPoint.tsx:13]: You can remove unused imports for readibility

**Review by louischristner — CHANGES_REQUESTED** [review]
Good job, it's looking quite noice!
Just take a look at my comments, nothing really important but I think it can improve overall readability on this component.

**Comment by hgvo9** [comment]
> 

Done with last commit

**Pull request merged**